addons/l10n_it_edi_doi/models/res_company.py

```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _name = 'res.company'
+    _inherit = 'res.company'
+
+    l10n_it_edi_doi_tax = fields.Many2one(
```

Comment:
```suggestion
    l10n_it_edi_doi_tax_id = fields.Many2one(
```
M2O field names should have an `_id` post-fix (same below) :wink: